qa/rpc-tests/wallet_dumpsigma.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+import time
+import os
+from decimal import *
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+from test_framework.test_helper import get_dumpwallet_otp
+
+#Scenario:
+#Generate 401 block to activate sigma
+#Mint sigmacoins of all denominations multiple 3
+#generate 6 blocks
+#spend sigma of all denominations
+#generate one block
+#Mint sigma of all denominations and not confirm them
+#getlistunspendsigma
+#getlistspendsigma
+#dumpwallet
+#importwallet
+#verify actual list unspendsigma
+#verify actual list spend sigma
+
+class WalletDumpSigmaTest(BitcoinTestFramework):
+    def __init__(self):
+        super().__init__()
+        self.num_nodes = 4
+        self.setup_clean_chain = False
+
+    def setup_nodes(self):
+        # This test requires mocktime
+        enable_mocktime()
+        return start_nodes(self.num_nodes, self.options.tmpdir)
+
+    def run_test(self):
+        getcontext().prec = 6
+        self.nodes[0].generate(401)
```

Comment:
Test framework already generated 200 blocks for us when `setup_clean_chain` is `False` so we can reduce this number fo `201`.

Reply:
Good point.